src/logger.ts

```diff
@@ -539,8 +546,10 @@ export class Logger {
   /**
    * Gets the underlying Bunyan logger.
    */
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-  public getBunyanLogger() {
+  // leave this typed as any to keep if from trying to export the type from the untyped bunyan module
+  // this prevents consumers from getting node_modules/@salesforce/core/lib/logger.d.ts:281:24 - error TS2304: Cannot find name 'Bunyan'.
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+  public getBunyanLogger(): any {
```

Comment:
see note--newer TS resulted in a different `lib/logger.d.t.s` -- it properly figured out the type but because we're using that weird bunyan fork we don't have actual types.